app/models/hooks/web_hook.rb

```diff
@@ -29,12 +29,15 @@ class WebHook < ActiveRecord::Base
   validates :url, presence: true,
                   format: { with: URI::regexp(%w(http https)), message: "should be a valid url" }
 
-  def execute(data)
+  def execute(data, hooks_scope)
     parsed_url = URI.parse(url)
     if parsed_url.userinfo.blank?
       WebHook.post(url,
                    body: data.to_json,
-                   headers: { "Content-Type" => "application/json" },
+                   headers: { 
+                    "Content-Type" => "application/json",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.